ml-testing-toolkit/values.yaml

```diff
@@ -109,147 +109,6 @@ connection-manager-frontend:
   #       port: 5060
   #       paths: ['/']
 
-mysql:
-  ## repo and documentation https://github.com/helm/charts/tree/master/stable/percona-xtradb-cluster
-
-  ## disable when connection-manager-backend is disabled
-  enabled: false
-  # # For Hosted Mode
-  # enabled: true
-  image:
-    repository: "percona/percona-xtradb-cluster"
-    tag: "5.7.19"
-    pullPolicy: IfNotPresent
-
-  nameOverride: mysql
-
-  replicas: 1
-
-  mysqlUser: mcm
-  mysqlPassword: mcm
-  mysqlDatabase: mcm
-  allowRootFrom: "%"
-
-  persistence:
-    enabled: false
-    accessMode: ReadWriteOnce
-    size: 8Gi
-
-  nodeSelector: {}
-
-  podAnnotations: {}
-
-  tolerations: []
-
-  configFiles:
-    node.cnf: |+
-      [client]
-      default-character-set=utf8mb4
-
-      [mysql]
-      default-character-set=utf8mb4
-
-      [mysqld]
-      collation-server = utf8mb4_unicode_ci
-      init-connect='SET NAMES utf8mb4'
-      character-set-server = utf8mb4
-      datadir=/var/lib/mysql
-      default_storage_engine=InnoDB
-      binlog_format=ROW
-      innodb_flush_log_at_trx_commit  = 1
-      innodb_flush_method             = O_DIRECT
-      innodb_file_per_table           = 1
-      innodb_autoinc_lock_mode=2
-      bind_address = 0.0.0.0
-      wsrep_slave_threads=2
-      wsrep_cluster_address=gcomm://
-      wsrep_provider=/usr/lib/galera3/libgalera_smm.so
-      wsrep_cluster_name=galera
-      wsrep_sst_method=xtrabackup-v2
-      pxc_strict_mode=DISABLED
-      max_connections=999
-      max_user_connections=994
-      thread_cache_size=999
-
-  # metricsExporter:
-  #   enabled: true
-  #   tag: v0.11.0
-  #   commandOverrides: []
-  #   argsOverrides: []
-
-  # logTail: true
-
-  service:
-    type: ClusterIP
-    port: 3306
-    externalIPs: []
-
-
-mongodb:
```

Comment:
Take note that you did not remove the MongoDB dependency as per my comment above.

Reply:
As discussed, revert config as this will be done/tested as part of https://github.com/mojaloop/project/issues/3091